cypress/e2e/signIn.cy.js

```diff
@@ -18,21 +18,30 @@ describe('Sign In page', () => {
   });
 
   it('should provide an ability to log in with existing credentials', () => {
+    cy.register();
     signInPage.visit();
-    cy.register(user.email, user.username, user.password);
-
+    
     signInPage.emailField
-      .type(user.email);
+      .type('riot@qa.team');
     signInPage.passwordField
-      .type(user.password);
+      .type('12345Qwert!');
     signInPage.signInBtn
       .click();
 
     homePage.usernameLink
-      .should('contain', user.username);
+      .should('contain', 'riot');
```

Comment:
The username 'riot' is hardcoded. You should consider setting it as a variable for better maintainability of the tests.